### DISCUSSION TITLE
adding dynamic IPs of pgbouncer to pgwatch2

### DISCUSSION THREAD

**Discussion by AnirudhJadhav-toddleapp** [cybertec-postgresql/pgwatch2#776] [Q&A]
Hello,
I am able to monitor pgbouncer instance individually using their IPs and i can view those instances on grafana. But my current setup has autoscaling group which make n number of pgbouncer instances based on connection load (there is also load balancer). Is it possible to monitor those pgbouncers created from autoscaling group? Since i cannot individually insert those IPs in the pgwatch2 interface. 

<img width="1440" alt="Screenshot 2024-05-09 at 12 18 13 PM" src="https://github.com/cybertec-postgresql/pgwatch2/assets/143793836/9bc9c840-177e-4b0d-a2b4-c2c715c2f1c4">

**Comment by pashagolub** [comment]
Hey. You can directly update monitored sources yourself. Pgwatch will reread confusion and will add/remove updated entries

**Reply by AnirudhJadhav-toddleapp** [reply]
do you mean that i need to change pgwatch ec2 instance config to resolve my issue?
Sorry i didnt understand, can you elaborate? I hope you have understood my question of monitoring multiple pgbouncer instances created from autoscaling group. I cannot add manually those instances IPs in pgwatch2 because they will be static IPs of instances. 
can you provide any documentation link ?

**Reply by pashagolub** [reply]
You can add rows for monitored sources to the configuration database manually or with some special script/program.

https://pgwatch2.readthedocs.io/en/latest/custom_installation.html#custom-installation
